pyvo/registry/regtap.py

```diff
@@ -194,6 +178,137 @@ def getrecord(self, index):
         """
         return RegistryResource(self, index)
 
+    def to_table(self):
+        """
+        returns a brief overview of the matched results as an astropy table.
+
+        This is mainly intended for interactive use, where people would
+        like to inspect the matches in, perhaps, notebooks.
+        """
+        return table.Table([
+                list(range(len(self))),
+                [r.short_name for r in self],
+                [r.res_title for r in self],
+                [r.res_description for r in self],
+                [", ".join(sorted(r.access_modes())) for r in self]],
```

Comment:
I see that the rational may be to make the returned table as simple as possible, and as a user, I may also be interested in the `ivoid` in the returned table. Is it possible to add it?